dockerfiles/init/manifests/che.env

```diff
@@ -506,6 +507,10 @@ CHE_SINGLE_PORT=false
 # OpenShift infrastructure ignores this property because it uses Routes instead of ingresses.
 #CHE_INFRA_KUBERNETES_INGRESS_ANNOTATIONS__JSON=NULL
 
+# Creates Ingresses with Transport Layer Security (TLS) enabled
+# In OpenShift infrastructure, Routes will be TLS-enabled
+CHE_INFRA_KUBERNETES_TLS_ENABLED=false
```

Comment:
Maybe it makes sense to add `CHE_INFRA_KUBERNETES_TLS__SECRET` here.

Reply:
done